opennms-webapp/src/main/webapp/admin/index.jsp

```diff
@@ -133,6 +133,7 @@
         <ul class="list-unstyled mb-0">
             <li><a href="admin/sysconfig.jsp">System Configuration</a></li>
             <li><a href="admin/userGroupView/index.jsp">Configure Users, Groups and On-Call Roles</a></li>
+            <li><a href="admin/endpoint/index.jsp">Configure Grafana Endpoints (Reports only)</a></li>
```

Comment:
Can we bring this down on the page? Maybe in "Additional Tools" on in a new section.

Doesn't seem important or accessed frequently enough to be at the top.

Reply:
Done.